gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -435,6 +435,8 @@ def run(argv=None):
   # See https://issues.apache.org/jira/browse/BEAM-2801.
   header_fields = vcf_header_parser.get_vcf_headers(
       known_args.representative_header_file)
+  beam_pipeline_options = pipeline_options.PipelineOptions(pipeline_args)
```

Comment:
Any specific reason why you moved this line? I prefer its previous spot because `beam_pipeline_options` object is used exactly in the next line.

Reply:
Artifact of old implementation. Reverted.